.github/workflows/test.main.kts

```diff
@@ -67,7 +69,20 @@ workflow(
         runsOn = UbuntuLatest,
     ) {
         uses(action = Checkout())
-        run(name = "Check for all actions") {
+        run(
+            name = "Determine changed typings in pull requests",
+            command = """
+                if [ -n "${expr { github.base_ref }}" ]; then
+                    git fetch origin ${expr { github.base_ref }}
+                    {
+                        echo 'changed_typings<<EOF'
+                        git diff --name-only --diff-filter d --no-renames origin/${expr { github.base_ref }} ${expr { github.sha }} -- typings/**/action-types.yml typings/**/action-types.yaml
```

Comment:
If we add `base_ref` to the GitHub context available in the Kotlin-based steps, will it be easy to port there? I think jgit should make it.

Reply:
Even without having it in that context, you can give the `base_ref` also as env variable easily using `env = mapOf("base_ref" to expr { github.base_ref })`.

I thought about using JGit, but I knew exactly what commandline call gives exactly the information we need, so I went that way. But it can most probably be ported to JGit usage too, yes.

Reply:
Thanks for using KGit 🙇